app/views/carts/show.html.erb

```diff
@@ -1,39 +1,36 @@
 <div class="container">
-  <% @items.each do |item,quantity| %>
   <div class="panel-body">
-    <div class="row">
+    <% @items.each do |item,quantity| %>
+    <div class="row" id="item-<%= item.id %>">
       <div class="col-xs-2"><img class="img-responsive" src=<%= item.image %> >
       </div>
       <div class="col-md-4">
         <h4 class="product-name"><strong> <%= item.name %> </strong></h4>
         <h4><small> <%= item.description %> </small></h4>
         <h4><small> <strong>Merchant: </strong><%= item.user.name %> </small></h4>
       </div>
-      <div class="col-md-4">
-          <p><strong>Price:</strong>  <%= number_to_currency(item.price)%></p>
-          <p><strong>Quantity:</strong> <%= quantity %></p>
-          <p><strong>Subtotal:</strong> <%= number_to_currency(item.subtotal(quantity)) %></p>
-
+      <div class="col-md-3">
```

Comment:
Adjusted grid width here slightly to allow buttons to display on right side if larger display is being used.

<img width="1124" alt="screen shot 2019-02-17 at 8 29 14 pm" src="https://user-images.githubusercontent.com/41562392/52926428-ecede980-32f2-11e9-9c8f-0314235ef649.png">

Reply:
That makes sense, thanks!